src/cookieSync.js

```diff
@@ -18,9 +20,33 @@ function triggerPixel(url) {
   img.src = url;
 }
 
+function insertUserSyncIframe(url) {
+  let iframe = document.createElement('iframe');
+  iframe.src = url;
+  insertElement(iframe);
+};
+
+function insertElement(elm, doc, target) {
```

Comment:
this function is available in util, use one from util

Reply:
Done